### PR TITLE
XW-4315 | Update play icon for recirculation, cleanup recirculation

### PR DESCRIPTION
**Pull request by rybmat**
https://wikia-inc.atlassian.net/browse/XW-4315

* updated play icon
* removed unused code

@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/Recirculation/templates/client/premiumRail.mustache [hunk 1]
```diff
@@ -17,6 +17,19 @@
 						{{#thumbnail}}
 							<img src="{{thumbnail}}" alt="{{title}}" itemprop="thumbnail">
 						{{/thumbnail}}
+						{{#isVideo}}
+							<div class="video-icon-container">
+								<svg id="wds-player-icon-play" class="wds-player-icon-play-tiny" viewBox="0 0 180 180" width="100%" height="100%">
```
Comment (rybmat) [inline extensions/wikia/Recirculation/templates/client/premiumRail.mustache]: better ideas about how to include wds svg into mustache template rendered by js are very welcome

Comment (latata) [inline extensions/wikia/Recirculation/templates/client/premiumRail.mustache]: remove `id` as it won't be unique

File: extensions/wikia/Recirculation/styles/mixed-content-footer.scss [hunk 2]
```diff
@@ -275,36 +275,19 @@ h1.mcf-header {
...
-
-	&__video-icon {
-		$play-button-icon-small: 23px;
-		fill: $wds-color-white;
-		margin-left: 6px;
 
 		@media #{$wds-breakpoint-m-down} {
-			height: $play-button-icon-small;
-			width: $play-button-icon-small;
+			.wds-player-icon-play-large {
```
Comment (latata) [inline extensions/wikia/Recirculation/styles/mixed-content-footer.scss]: As we discussed offline - create mixin in DS and use it here.

File: extensions/wikia/Recirculation/templates/client/premiumRail.mustache (hunk 1 above)
Comment (latata) [reply extensions/wikia/Recirculation/templates/client/premiumRail.mustache]: I'd keep it as variable in JS and pass it to templates

**Review by latata — COMMENTED** [review]
is `recirculation-unit` class in `premiumRail.mustache` still needed?

**Review by latata — APPROVED** [review]

**Pull request merged**